manifests/0000_31_cluster-baremetal-operator_02_metal3provisioning.crd.yaml

```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     capability.openshift.io/name: baremetal
+    controller-gen.kubebuilder.io/version: ""
```

Comment:
Not sure if it's this line or not, but [generate-check][1] is complaining with the not-all-that-clear:

```
make[1]: Leaving directory '/go/src/github.com/openshift/cluster-baremetal-operator'
3c3
< 234[61](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-baremetal-operator/272/pull-ci-openshift-cluster-baremetal-operator-master-generate-check/1539250566973100032#1:build-log.txt%3A61)12792 13356 manifests/0000_31_cluster-baremetal-operator_02_metal3provisioning.crd.yaml
---
> 2882206592 13310 manifests/0000_31_cluster-baremetal-operator_02_metal3provisioning.crd.yaml
make: *** [Makefile:131: generate-check] Error 1
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-baremetal-operator/272/pull-ci-openshift-cluster-baremetal-operator-master-generate-check/1539250566973100032